src/Helpers/ObjectHelperTrait.php

```diff
@@ -38,29 +38,29 @@ public function isValidXml(string $xml)
 	/**
 	 * Check if json is valid
 	 *
-	 * @param string $string String to check.
+	 * @param string $check String to check.
 	 *
 	 * @return bool
 	 */
-	public static function isJson(string $string): bool
+	public static function isJson(string $check): bool
```

Comment:
```suggestion
	public static function isJson(string $jsonString): bool
```

Is more clear when you type `isJson(jsonString: '....')`. Although I don't think this is needed since it's just one argument so maybe we could just leave it 🤷🏼‍♂️